datalad/distribution/tests/test_clone.py

```diff
@@ -309,3 +310,13 @@ def test_clone_isnt_a_smartass(origin_path, path):
     assert clonedsub.path.startswith(path)
     # no subdataset relation
     eq_(cloned.subdatasets(), [])
+
+
+@skip_if_on_windows
+def test_clone_report_permission_issue():
+    with chpwd('/'):
```

Comment:
please do not!  noone forbids running datalad or its tests as root, which we do during package building.  This obviously would fail and leave junk laying around.  IMHO regular `with_tempfile(mkdir=True)` + mkdir + chmod call within and then chpwd into it would be sufficient BUT still would require skipping the test if ran as super-user -- there user/git has too much power 
```shell
$> mkdir -p /tmp/testperm/d; chmod a-w /tmp/testperm/d; cd /tmp/testperm/d; datalad install ///
[INFO   ] Cloning dataset from 'http://datasets.datalad.org/' (trying 2 location candidate(s)) to '/tmp/testperm/d/datasets.datalad.org' 
[ERROR  ] Failed to clone data from any candidate source URL: ['http://datasets.datalad.org/', 'http://datasets.datalad.org/.git'] [install(/tmp/testperm/d/datasets.datalad.org)] 
install(error): /tmp/testperm/d/datasets.datalad.org (dataset) [Failed to clone data from any candidate source URL: ['http://datasets.datalad.org/', 'http://datasets.datalad.org/.git']]
```
and